doc/man1/openssl-asn1parse.pod.in

```diff
@@ -41,7 +41,7 @@ Print out a usage message.
 =item B<-inform> B<DER>|B<PEM>
 
 The input format; the default is B<PEM>.
-See L<openssl(1)/Format Options> for details.
+See L<openssl-format-options(1)/Format Options> for details.
```

Comment:
I don't see the need to include the section.  `L<openssl-format-options(1)>` should suffice.

Reply:
This applies to *every* such link you changed :wink: